frontend/src/app/pages/item-create/item-create.component.html

```diff
@@ -3,47 +3,53 @@ <h1>Creating new item</h1>
 </div>
 
 <div class="container">
-  <form [formGroup]="form"  (ngSubmit)="onSubmit()">
-    <mat-form-field appereance = "fill">
+  <form [formGroup]="form" (ngSubmit)="onSubmit()">
+    <mat-form-field appereance="fill">
       <mat-label>Name</mat-label>
-      <input matInput maxlength="15" inputmode="tel" placeholder="Type new name" formControlName="name" #nameInput>
+      <input matInput maxlength="255"
+             inputmode="tel"
```

Comment:
`inputmode="tel"` is a telephone keypad input. I think you can remove this completely. You have this in several places across project.